rfi_file_monitor/operations/url_downloader.py

```diff
@@ -1,104 +0,0 @@
-import requests
```

Comment:
You can probably drop `requests` from the list of dependencies when removing this file, even though it may still get dragged in by one of the remaining ones.

Reply:
it is used in the googleanalytics part I think so I have left it for now